cpp/src/Slice/Parser.cpp

```diff
@@ -606,6 +608,10 @@ Slice::DocComment::parseFrom(const ContainedPtr& p, DocLinkFormatter linkFormatt
                 }
             }
         }
+        else if (parseCommentLine(line, remarkTag, lineText) || parseCommentLine(line, remarksTag, lineText))
+        {
+            currentSection = &(comment._remarks);
```

Comment:
Since we have earlier:
```cpp
StringList* currentSection = &comment._overview;
```

it's likely all the parenthesis are superfluous.

Reply:
Indeed, I removed all the parenthesis.